src/core/lua/luaadapter.cpp

```diff
@@ -24,7 +24,6 @@ CLuaAdapter::~CLuaAdapter() = default;
 
 bool CLuaAdapter::initialize() {
   //before calling super we need to configure the interface of the FB
-  setupFBInterface(getFBInterfaceSpec());
   return CAdapter::initialize();
```

Comment:
as we are now only calling super we could remove the whole method and with that make the code cleaner.

Reply:
done